UDV-Core/src/Modules/ConsultDoc/DocumentController.js

```diff
@@ -0,0 +1,142 @@
+/**
+ * Class: DocumentController
+ * Description :
+ * The DocumentController is an object handling the views, interracting with the
+ * server to get information and data (documents)
+ *
+ */
+
+import { DocumentResearch }  from './DocumentResearch.js';
+import { DocumentBrowser }   from './DocumentBrowser.js';
+import './ConsultDoc.css';
+
+/**
+ * Constructor for DocumentController Class
+ * @param controls : PlanarControls instance
+ * @param options : optional parameters (including TemporalController)
+ * @param view :  itowns planar view
+ * @param docModel : file holding document model
+ */
+//=============================================================================
+export function DocumentController(view, controls, options = {},docModel, researchModel,optionsResearch)
+{
+
+    this.url = "http://rict.liris.cnrs.fr/APIVilo3D/APIExtendedDocument/web/"; //url of the server handling documents
+    //FIXME: to put in a configuration file of the general application
```

Comment:
I just created issue #14 to get this fixed in the next PR.
Issues are much more "sticky" (better reminders) than FIXMEs hidden in the code.